projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -31,6 +31,8 @@
       [class]="node.styleClasses"
       (click)="closeIfClickedTheSameLink(node)"
     >
+      <cx-icon *ngIf="getIcon(node.title)" [type]="getIcon(node.title)">
+      </cx-icon>
```

Comment:
[MEDIUM]
I believe this component was meant for the top navigation of the storefront. Are you sure we'd like to add here an option fur supporting icons?

Reply:
this component is being used for all the top and footer navigation...please suggest if i have to create a new component for the same